repository/src/main/java/org/apache/atlas/transformer/PreProcessorPoliciesTransformer.java

```diff
@@ -0,0 +1,130 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.atlas.transformer;
+
+import org.apache.atlas.exception.AtlasBaseException;
+import org.apache.atlas.model.instance.AtlasEntity;
+import org.apache.atlas.model.instance.AtlasEntity.AtlasEntitiesWithExtInfo;
+import org.apache.atlas.repository.store.graph.v2.preprocessor.ConnectionPreProcessor;
+import org.apache.atlas.type.AtlasType;
+import org.apache.commons.collections.MapUtils;
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.apache.atlas.repository.Constants.NAME;
+import static org.apache.atlas.repository.Constants.QUALIFIED_NAME;
+import static org.apache.atlas.repository.Constants.getStaticFileAsString;
+import static org.apache.atlas.repository.util.AccessControlUtils.ATTR_POLICY_RESOURCES;
+import static org.apache.atlas.repository.util.AccessControlUtils.ATTR_POLICY_ROLES;
+import static org.apache.atlas.repository.util.AccessControlUtils.getPolicyRoles;
+import static org.apache.atlas.repository.util.AtlasEntityUtils.getListAttribute;
+import static org.apache.atlas.repository.util.AtlasEntityUtils.getName;
+import static org.apache.atlas.repository.util.AtlasEntityUtils.getQualifiedName;
+
+public class PreProcessorPoliciesTransformer {
+    private static final Logger LOG = LoggerFactory.getLogger(ConnectionPreProcessor.class);
```

Comment:
since connection/collection preprocessors are not marked as @component, here PreProcessorPoliciesTransformer is being initialized using new keyword, can not make constructor private

Will mark class as final